env/build_jet_intel.env_DA

```diff
@@ -0,0 +1,36 @@
+#Setup instructions for NOAA RDHPC Jet using Intel-18.0.5.274 (bash shell)
+#source /etc/profile.d/modules.sh
+
+module purge
+module load intel/18.0.5.274
+module load impi/2018.4.274
+module load szip/2.1 hdf5/1.8.9 netcdf/4.2.1.1
+module load pnetcdf/1.6.1
+module load nco/4.1.0
+module load cnvgrib/1.4.0
+module load cmake/3.16.1
+
+NCEPLIBS="/lfs4/BMC/wrfruc/gge/precompiled/NCEPLIBS/b_intel18.0.5.274_impi2018.4.274/install"
+GSILIBS="/lfs4/BMC/wrfruc/gge/precompiled/GSILIBS/b_intel18.0.5.274_impi2018.4.274"
```

Comment:
Why do these need to be loaded in this way instead of the way they are currently loaded in the build Jet env file?

Reply:
This is not a good time for GSI environment development. Will look for better compiling solution later.